website/documentation/content/user_documentation.py

```diff
@@ -156,6 +156,40 @@ def receive_file(e: events.UploadEventArguments):
             ''')
 
 
+doc.text('Test Downloads', '''
+    You can verify that a download was triggered by checking `user.downloads.http_responses`.
+    By awaiting `user.downloads.next()` you can get the next download response.
```

Comment:
This says `next`, but the method's name is "new"...?

(personally, I like "next")

Reply:
Yes, we agree. I was just about to commit a fix. 👍🏻